npe2/manifest/schema.py

```diff
@@ -26,6 +27,27 @@
 NPE1_ENTRY_POINT = "napari.plugin"
 
 
+class Category(str, Enum):
+    """Broad plugin categories, values for PluginManifest.categories."""
+
+    Acquisition = "Acquisition"
+    Annotation = "Annotation"
+    Data = "Data"
+    Denoising = "Denoising"
+    Image_Processing = "Image Processing"
+    IO = "IO"
+    Machine_Learning = "Machine Learning"
+    Measurement = "Measurement"
+    Segmentation = "Segmentation"
+    Themes = "Themes"
+    Transformations = "Transformations"
+    Utilities = "Utilities"
+    Visualization = "Visualization"
+
```

Comment:
```suggestion
    Simulation = "Simulation"
    Just_For_Fun = "Just for Fun"
    
```

I often use napari as a simulation playground and there are quite a few plugins that are 'just for fun' type things :-)

Reply:
i like simulation!  Not as sure about just for fun :)  I know robert has a number of just for fun but otherwise useless plugins... but i'm not sure they warrant their own category?

Reply:
Fair! I could go either way but ultimately don't mind :)